config/core/configmaps/deployment.yaml

```diff
@@ -43,3 +43,7 @@ data:
 
     # List of repositories for which tag to digest resolving should be skipped
     registriesSkippingTagResolving: "ko.local,dev.local"
+
+    # ProgressDeadlineSeconds is the time in seconds we wait for the deployment to
+    # be ready before considering it failed.
+    progressDeadline: "120"
```

Comment:
This should either be a parseable duration or contain the word `Seconds` in it. I think other config maps use durations here?

Reply:
Sounds good. Converted.